src/methods/saveData.ts

```diff
@@ -0,0 +1,57 @@
+import log from "../helpers/log.js"
+import { SimpleDataItem, Options, defaultOptions } from "../types.js"
+import showTable from "./showTable.js"
+//@ts-ignore
+
+export default async function saveData(data: SimpleDataItem[], path: string, options: Options) {
+
+    const start = Date.now()
+
+    options = {
+        ...defaultOptions,
+        ...options
+    }
+
+    options.logs && log("\saveData() " + path)
+    options.logOptions && log("options:")
+    options.logOptions && log(options)
+
+    if (options.environment === "nodejs") {
+
+        const fs = await import("fs")
+
+        const pathSplit = path.split(".")
+        const extension = pathSplit[pathSplit.length - 1].toLowerCase()
+
+        if (extension === "csv") {
```

Comment:
I would suggest splitting this into `toJSON` and `toCSV`. It's going to be easier to reuse, test and catch potential bugs.

Reply:
I would keep it like that for the moment. It's so simple to use saveData() directly! :)